resin/auth.py

```diff
@@ -119,6 +148,12 @@ def get_token(self):
         Raises:
             InvalidOption: if not logged in and there is no token in Settings.
 
+        Examples:
+            # If you are logged in.
```

Comment:
You're right, but the docs-gen code doesn't seem to figure out this is actually the start of a code block, thus rendering it weirdly which can be fixed by prepending ">>>". We should either fix the docs generator (which is preferrable), or work around it :)